src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.MarketplaceOrdering</PackageId>
+    <Description>Provides developers with a library to get and set legal terms for Azure Marketplace Offers. Note: This client library is for MarketplaceOrdering under Azure Resource Manager.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Also as this is your first version, is your service GA already? If not then mark your version as 1.0.0-preview

Reply:
REST API is already in prod, the swagger is being merged into public repo.